fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1125,6 +1129,12 @@ public void setEnableParallelMerge(boolean enableParallelMerge) {
     @VarAttr(name = ENABLE_PLAN_VALIDATION, flag = VariableMgr.INVISIBLE)
     private boolean enablePlanValidation = true;
 
+    @VarAttr(name = SCAN_OR_TO_UNION_LIMIT, flag = VariableMgr.INVISIBLE)
+    private int scanOrToUnionLimit = 4;
```

Comment:
How many scan operator for the or 4 Children

Reply:
One scan can split to at most 4 union.